lib/modules.nix

```diff
@@ -1006,6 +1033,8 @@ let
       map (mapAttrs (n: v: mkIf cfg.condition v)) (pushDownProperties cfg.content)
     else if cfg._type or "" == "override" then
       map (mapAttrs (n: v: mkOverride cfg.priority v)) (pushDownProperties cfg.content)
+    # else if cfg._type or "" == "definition" then
+    #   map (mapAttrs (n: v: mkDefinition v)) (pushDownProperties cfg.content)
```

Comment:
What's up with this?

Reply:
Got removed due to rebasing, we resolved this in the mkDefinition PR.